### PR TITLE
Tmedia 468 header nav chain buttons 

### PR DESCRIPTION
**Pull request by JackHowa**
## Description
Header nav chain buttons, querly, arc-search, sections

## Jira Ticket
- [TMEDIA-468](https://arcpublishing.atlassian.net/browse/TMEDIA-468)

## Acceptance Criteria

- Match color logic for nav color and borders for sections, queryly, arc search (not expanded)
- Expanded input should always have white background and incorporate the button similarly as before

## Test Steps

1. Checkout this branch `git checkout TMEDIA-468-header-nav-chain-buttons`
2. Run fusion repo with linked blocks `npx fusion start -f -l  @wpmedia/identity-block,@wpmedia/shared-styles,@wpmedia/header-nav-chain-block`
3. Create homepage with sections, queryly on the left and arc search on the right area
4. Show the page on all three potential nav color iterations the-sun (primary-color), dagen (light), and the-prophet (dark) 
5. Should match color effects based on mocks 

Screen Shot 2021-09-27 at 15 01 19

### Before
- Dark and light color only 
- Used inconsistent border color 

### After
Screen Shot 2021-09-27 at 15 03 30

## Dependencies or Side Effects
- Shared styles scss has been changed

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
please explain why (so that we can fix it whenever it gets refactored). -> removed theme color params to use properties
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

**Comment by matthewroach** [comment]
Search button is a different size to the menu button - these should be the same

Microsoft Edge-2021-09-28-17-26 21

**Comment by JackHowa** [comment]
updated -- thanks @matthewroach -- and wrote sb stories to ensure no changes 

Screen Shot 2021-09-28 at 13 34 52

File: blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/search-box.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.search-box--right-absolute-positioned {
```
Comment (matthewroach) [inline blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/search-box.scss:1]: Looks like this is causing issues when you have the search box configured on the left side

Existing
Microsoft Edge-2021-09-29-11-26 36

With this change -
Microsoft Edge-2021-09-29-11-26 53

**Review by matthewroach — CHANGES_REQUESTED** [review]
Small issue with search when on the left side

Also, I do not see a story for two button side by side?

File: blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/search-box.scss (hunk 1 above)
Comment (JackHowa) [reply blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/search-box.scss:1]: thanks @matthewroach -- updated. only displaying on open now

Screen Shot 2021-09-29 at 09 21 48
Screen Shot 2021-09-29 at 09 21 40

**Comment by JackHowa** [comment]
@matthewroach storybook updated with two buttons next to each other 

image

**Review by matthewroach — APPROVED** [review]

**Pull request merged**